addons/l10n_in_edi_ewaybill/models/error_codes.py

```diff
@@ -321,4 +330,7 @@
     "817": _lt("MultiVehicleMovement cannot be initiated.Eway Bill is not in Active State"),
     "818": _lt("Validity period lapsed.Cannot generate consolidated Eway Bill"),
     "819": _lt("Ewaybill cannot be generated for the document date which is prior to 01/07/2017"),
+    "820": _lt("You cannot generate E Way Bill with document date earlier than 180 days"),
+    "821": _lt("E Way Bill cannot be extended as allowed limit is 360 days"),
+    "4043": _lt("You cannot generate E Way Bill with document date earlier than 180 days"),
```

Comment:
```suggestion
    "4043": _lt("You cannot generate Ewaybill with document date earlier than 180 days"),
```
It is possible to do something consistent for how Ewaybill is written?